src/Entity/OgMembership.php

```diff
@@ -320,6 +320,22 @@ public function preSave(EntityStorageInterface $storage) {
       throw new \LogicException('OG membership can not be created for an empty or anonymous user.');
     }
 
+    if (!$this->get('entity_id')->value) {
+      // Group was not set.
+      throw new \LogicException('Membership cannot be set for an empty or an unsaved group.');
```

Comment:
Regarding the first issue seems that the syntax in the docs is wrong, I'm going to change it.

`->setConstraints(['Range' => ['min' => 5]]);`
or
`->addConstraint('Range', ['min' => 5]);`

But still not clear on the second error where I can't do `$membership->validate()`